lib/widgets/trufi_drawer.dart

```diff
@@ -26,12 +26,14 @@ class TrufiDrawer extends StatefulWidget {
 
 class TrufiDrawerState extends State<TrufiDrawer> {
   AssetImage bgImage;
-  final GlobalKey appShareButtonKey = GlobalKey(debugLabel: "appShareButtonKey");
+  final GlobalKey appShareButtonKey =
+      GlobalKey(debugLabel: "appShareButtonKey");
 
   @override
   void initState() {
     super.initState();
 
+    // TODO: Should have some kind of fallback image
     bgImage = AssetImage(TrufiConfiguration().image.drawerBackground);
```

Comment:
I suggest we don't set an image then. It will have the primary color as the background then.

Reply:
Yes, I thought the same, but for the moment, I did not want to change any more logic. It would be a separate PR where I go through all the TODO´s I created.
I could also change the Todo text if you like.